src/EnergyPlus/UnitarySystem.hh

```diff
@@ -409,6 +409,8 @@ namespace UnitarySystems {
         bool m_OKToPrintSizing;
         bool m_IsDXCoil;
         Real64 m_SmallLoadTolerance;
+        bool m_waterSideEconomizerFlag;   // true if water-side economizer coil is active
+        Real64 m_minAirToWaterTempOffset; // coil entering air to entering water temp offset
```

Comment:
Are you suggesting that if the offset is zero, then it is a regular cooling coil?

Reply:
Yes and no. From what I could see, the only special about a "waterside economizer" coil is the the temperature offset control is active. If the offset is zero, that control does nothing. Also, this offset could be used for a normal coil, if desired. So, it seems the real question here is whether the temperature offset control is active or not. But maybe I'm missing something.

Reply:
OK. Now the var ```m_TemperatureOffsetControlActive``` is set true ```if(m_minAirToWaterTempOffset > 0.0)```, and the model assumes that it is a water side economizer regardless of plant loop type the Coilsystem is connected to; otherwise, it is just a regular water cooling coil. This makes the new Coilsystem object be used for any cooling operation provided the user places/connects it correctly for the intended application.